src/logcollector/main.c

```diff
@@ -104,26 +101,28 @@ int main(int argc, char **argv)
         merror_exit(SETGID_ERROR, group, errno, strerror(errno));
     }
 
+    /* Read config file */
+    if (LogCollectorConfig(cfg) < 0) {
+        merror_exit(CONFIG_ERROR, cfg);
+    }
+
+    sock_fail_time = log_config.sock_fail_time;
```

Comment:
Why do you need this variable?

Reply:
This variable is read here
https://github.com/wazuh/wazuh/blob/86ccb6cd55511e5d9727c8a8bfcd0465720e6371/src/shared/mq_op.c#L151
The configuration structure cannot be accessed from that shared file.